kolibri/core/content/api.py

```diff
@@ -1382,10 +1383,29 @@ class UserContentNodeViewset(BaseContentNodeMixin, BaseValuesViewset, ListModelM
     A content node viewset for filtering on user specific fields.
     """
 
-    filter_backends = (DjangoFilterBackend,)
+    filter_backends = (DjangoFilterBackend, filters.OrderingFilter)
+    ordering_fields = ["last_interacted"]
+    ordering = ("lft", "id")
     filter_class = UserContentNodeFilter
     pagination_class = OptionalPagination
 
+    def get_queryset(self):
+        user = self.request.user
+        # if user is anonymous, don't return any nodes
```

Comment:
Based off this comment, if the user is anonymous, should this return `models.ContentNode.objects.none()`?
https://docs.djangoproject.com/en/3.2/ref/models/querysets/#none

Reply:
I think this was might be a copy paste remnant from the `get_resume_queryset` function - so should just be deleted.